packages/typespec-ts/test/modularUnit/scenarios/apiOperations/apiOperations.md

```diff
@@ -703,15 +704,23 @@ export function _testSend(
   apiVersion: string,
   options: TestOptionalParams = { requestOptions: {} },
 ): StreamableMethod {
+  const path = expandUrlTemplate(
+    "/{?api-version}",
+    {
+      "api-version": apiVersion,
+    },
+    {
+      allowReserved: options?.requestOptions?.skipUrlEncoding,
+    },
+  );
   return context
-    .path("/")
+    .path(path)
     .get({
       ...operationOptionsToRequestParameters(options),
       headers: {
         accept: "application/json",
         ...options.requestOptions?.headers,
       },
-      queryParameters: { "api-version": apiVersion },
```

Comment:
is there a case where headers parameters use url template? how do we support this? I remember previously typespec hasn't supported it but I think Timothee has added that support in typespec now.

Reply:
header is not supported yet and issue tracked here: https://github.com/Azure/autorest.typescript/issues/3095.